pages/works/index.tsx

```diff
@@ -5,6 +5,8 @@ import Layout from "@/components/layout";
 import { InView } from "react-intersection-observer";
 import { map as lodashMap, groupBy as lodashGroupBy } from "lodash";
 import Grid from "@/components/Grid/Grid";
+import Metatag from "../../components/Metatag/Metatag";
+import { dev } from "@/canopy.config";
```

Comment:
Should this be `dev` or `prod`? Wonder how we can make this environment based in a smarted way?

Reply:
I wonder if it should just be its own section in canopy config and not be in dev or prod.  I definitely feel like `prod` makes more sense.

Reply:
Should we handle this now or tackle later?

Reply:
@mathewjordan Now.  Let's don't merge anything until it's right.